psi-ms.obo

```diff
@@ -21985,6 +21987,112 @@ is_a: MS:1003201 ! library provenance attribute
 xref: value-type:xsd:string "The allowed value-type for this CV term."
 relationship: has_value_type xsd\:string ! The allowed value-type for this CV term
 
+[Term]
+id: MS:1003207
+name: mass spectrometry mass selection/dissociation method aspect
+def: "Specific aspect of a mass spectrometer method by which mass ranges are selected and possibly dissociated." [PSI:MS]
+is_a: MS:1001458 ! spectrum generation information
+
+[Term]
+id: MS:1003208
+name: data-independent acquisition
+def: "Mass spectrometer data acquisition method wherein mass selection for fragmentation is configured according to a pre-determined program, rather than based on any detected precursor ions." [PSI:MS]
+synonym: "DIA" EXACT []
+is_a: MS:1003207 ! mass spectrometry mass selection/dissociation method aspect
+
+[Term]
+id: MS:1003209
+name: dissociation of full mass range
+def: "Mass spectrometer data acquisition method wherein all precursor ions of which the instrument is capable are fragmented at once.." [PSI:MS]
+is_a: MS:1003207 ! mass spectrometry mass selection/dissociation method aspect
+
+[Term]
+id: MS:1003210
+name: dissociation of scanning quadrupole across a specified mass range
+def: "Mass spectrometer data acquisition method wherein ????." [PSI:MS]
+is_a: MS:1003207 ! mass spectrometry mass selection/dissociation method aspect
+
+[Term]
+id: MS:1003211
+name: dissociation of sequential mass ranges
+def: "Mass spectrometer data acquisition method wherein a series of limited mass range fragmentation selection windows are preconfigured." [PSI:MS]
+is_a: MS:1003207 ! mass spectrometry mass selection/dissociation method aspect
+
+[Term]
+id: MS:1003212
+name: ion mobility separation
+def: "Mass spectrometer data acquisition method wherein precursor ions are separated by their ion mobility properties prior to measurement." [PSI:MS]
+is_a: MS:1003207 ! mass spectrometry mass selection/dissociation method aspect
+
+[Term]
+id: MS:1003213
+name: adduct deconvolution
+def: "Data processing action of merging of the measurements of potentially multiple adducts into a single representation that is independent of the small ion that adds charge to a larger molecule." [PSI:MS]
+is_a: MS:1000543 ! data processing action
+
+[Term]
+id: MS:1003214
+name: data-dependent acquisition
+def: "Mass spectrometer data acquisition method wherein MSn spectra are triggered based on the m/z of precursor ions detected in the same run." [PSI:MS]
+synonym: "DDA" EXACT []
+is_a: MS:1003216 ! mass spectrometry mass selection/dissociation method
+
+[Term]
+id: MS:1003215
+name: ion mobility deconvolution
+def: "Data processing action of merging multiple ion peaks acquired at different ion mobility steps into a single mass spectrum representing a single analyte." [PSI:MS]
+is_a: MS:1000543 ! data processing action
+
+[Term]
+id: MS:1003216
+name: mass spectrometry mass selection/dissociation method
```

Comment:
Not super happy with this name and the slash, but if SIM is going to be parented under it, the dissociation has to be optional.